.github/workflows/release.yml

```diff
@@ -93,6 +100,13 @@ jobs:
           ref=${{ github.head_ref || github.ref_name }}
           echo "release_version=${ref##*/}-$release_hash" >> $GITHUB_OUTPUT
         shell: bash
+      - name: Install aarch64 cross compilation prerequisites
+        if: ${{ matrix.platform == 'linux-aarch64' }}
+        run: |
+          sudo apt-get install -y cmake gcc g++ ninja-build libtool autoconf pkg-config gcc-13-aarch64-linux-gnu g++-13-aarch64-linux-gnu curl zip unzip tar
```

Comment:
Don't we also need the arm64 system libraries?

Reply:
I don't think so, but let's see what the workflow thinks